bin/dotd

```diff
@@ -170,16 +167,12 @@ def dotd_menu_prompt
 
         #{bold 'Q'}: #{underline 'Q'}uit.
 
-    Your Choice: 
+    Your Choice:#{' '}
```

Comment:
My editor auto-trims trailing whitespace. I suspect this is true of others as well. This seems like a more effective solution.